src/components/FolderPicker/FolderPickerContentRoot.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import { useQuery } from 'cozy-client'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+import { FolderPickerContentExplorer } from 'components/FolderPicker/FolderPickerContentExplorer'
+import { FolderPickerContentLoadMore } from 'components/FolderPicker/FolderPickerContentLoadMore'
+import { FolderPickerContentLoader } from 'components/FolderPicker/FolderPickerContentLoader'
+import { ROOT_DIR_ID } from 'constants/config'
+import { DumbFile as File } from 'modules/filelist/File'
+import { buildOnlyFolderQuery } from 'modules/queries'
+import { buildSharedDrivesQuery } from 'modules/views/Folder/queries/fetchSharedDrives'
+
+const FolderPickerContentRoot = ({ navigateTo }) => {
```

Comment:
This component seems to do quite a lot of things. and some of them seem reusable (fetchStatus). Maybe it's possible here to remove code from the component, into hooks or stateless functions